src/Validation/Validator.php

```diff
@@ -2247,6 +2731,13 @@ public function multipleOptions(
         ?string $message = null,
         Closure|string|null $when = null
     ) {
+        if ($message === null) {
+            $message = 'The provided value must be a set of multiple options';
+            if ($this->_useI18n) {
+                $message = __d('cake', 'The provided value must be a set of multiple options');
+            }
+        }
```

Comment:
This could mention the optons set: But this would lead to many code branches. Not sure if that's worth the effort.

https://github.com/cakephp/cakephp/blob/332baf6a13532e60c5537315edfdfe41c1a68eee/src/Validation/Validation.php#L944-L946